docs/releases.md

```diff
@@ -269,9 +269,12 @@ For good measures, check the console log to confirm that the correct release pac
 Adjust state and `Released As` of [Jira issues](https://issues.jenkins.io/) fixed in the release (see the [changelog](https://www.jenkins.io/changelog-stable) for issue links)
 
 ### Publish Docker image
-⚠️ **Requires access to the network trusted.ci.jenkins.io is in** ⚠️
+⚠️ **Requires write access to the repository jenkinsci/docker** ⚠️
```

Comment:
In case a hyperlink helps a reader understand what is being said...
```suggestion
⚠️ **Requires write access to the [jenkinsci/docker repository](https://github.com/jenkinsci/docker) ** ⚠️
```